eoscostarica/lifebank
# feature(user): add credentials recovery using email

### What does this PR do?
Add credentials recovery option
Resolve #58 

### How should this be tested?
- make run
- open sidebar menu
- check "Credentials recovery" option 
- enter a valid user email
